course4/homework1/test.cc

```diff
@@ -22,6 +22,9 @@ int main()
 	s << a;
 
 	assert(s.str() == "(5, 6)");
+	
+	double pi = 3.14159265358979323846;
+	std::cout << Vector2d(PolarCoords2d(5, pi/2)) << std::endl;
```

Comment:
This is not really a test. It just prints the value., and requires manual verification of its correctness.
Use ```assert``` to make the test fully automated.